src/benchmark_register.cc

```diff
@@ -159,6 +159,7 @@ bool BenchmarkFamilies::FindBenchmarks(
         instance.use_manual_time = family->use_manual_time_;
         instance.complexity = family->complexity_;
         instance.complexity_lambda = family->complexity_lambda_;
+        instance.statistics = &family->statistics_;
```

Comment:
Hmm. I'm using `-fsanitize=address -fsanitize=undefined`, and at least with all the current tests, there are no complaints.